wasm-chain-integration/src/types.rs

```diff
@@ -67,9 +67,9 @@ impl<Policies> ReceiveContext<Policies> {
     pub fn self_address(&self) -> &ContractAddress { &self.self_address }
 }
 
-pub(crate) fn deserial_receive_context<'a>(
-    source: &'a [u8],
-) -> ParseResult<ReceiveContext<&'a [u8]>> {
+pub(crate) fn deserial_receive_context(
+    source: &[u8],
+) -> ParseResult<ReceiveContext<&[u8]>> {
```

Comment:
Was this change needed?

I originally put the lifetimes there to be explicit, even though they can be elided.

Reply:
I got a warning about them from cargo clippy I believe, should I put them back?

Reply:
It's not a big deal so do what you think is best.

But is this using the nightly clippy? I think we do have CI setup to check this, and it does not complain with the version we use.